src/status_im/ui/screens/communities/reorder.cljs

```diff
@@ -0,0 +1,36 @@
+(ns status-im.ui.screens.communities.reorder
```

Comment:
Hi, do we need separate files for tabs view? Wouldn't it will be better if we just use this view as a function?